skimage/feature/_sift.pyx

```diff
@@ -0,0 +1,167 @@
+#cython: cdivision=True
+#cython: boundscheck=False
+#cython: nonecheck=False
+#cython: wraparound=False
+
+import numpy as np
+cimport numpy as cnp
+from libc.math cimport M_PI, floor
+
+from .._shared.fused_numerics cimport np_floats
+
+
+cpdef _ori_distances(np_floats[::1] ori_bins,
+                     np_floats[::1] theta):
+    """Compute angular minima and their indices.
+
+    Parameters
+    ----------
+    ori_bins : ndarray
+        The orientation histogram bins.
+    theta : ndarray
+        The orientation computed for each pixel in the patch.
+
+    Returns
+    -------
+    near_t : ndarray
+        The histogram bin index corresponding to each pixel in the patch.
+    near_t_val : ndarray
+        The distance between the orientation of each pixel and the nearest bin.
+    """
+    cdef:
+        Py_ssize_t n_theta = theta.size
+        Py_ssize_t n_ori = ori_bins.size
+        Py_ssize_t i, j, idx_min
+        double th, dist, dist_min
+        double two_pi = 2 * M_PI
```

Comment:
Shouldn't these `double` be `np_floats`?

Reply:
I agree, but it's probably more a question for @grlee77

Reply:
I fixed this, thanks. I think I initially started with just double and missed these when switching over to `np_floats`